src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+or more `wp-block-`-prefixed classes. These are the blocks documented below.
+
+Each block has controls that can modify the generated output. Where relevant, the
+additional markup and/or classes that may be generated are also documented.
+
+### Editor Controls
+
+Block editor controls vary, but many have Styles and Color settings. The effects
+of other controls are described in specific block stories.
+
+#### Styles
+
+These controls offer 2 or more options for particular stylistic variations, and
+the ability to set one as a default for new blocks of the same type. For example,
+images can have a circular mask or not. These options add an `is-style-`-type class
+to the block.
+
+#### Color settings
+
+These controls enable setting of the block's `background-color` and `color`.
+
+The base class for a block with a main (background) color is `has-background` and
+selecting a text color adds a `has-text-color` class. If one of a set of predefined
+colors is chosen, additional classes are added, such as `has-subtle-background-background-color`
+for `background-color` or `has-secondary-color` for text.
+
+If user-defined colors are chosen, instead, inline styles will set `color` and/or
+`background-color`.
+
+## Audio
+
+The audio block wraps an `audio` element and an optional `figcaption` element in
+a `figure` with class `wp-block-audio`
+
+<Preview>
+  <Story name="core/audio" height="150px">{`
+    <figure class="wp-block-audio type">
+      <audio controls="" src=""></audio>
+      <figcaption>My audio clip</figcaption>
+    </figure>
+  `}</Story>
+</Preview>
+
+## Button
+
+The button block creates one or more `div`s with class `wp-block-button`, each containing
+an `a` element with a class of `wp-block-button__link`.
+
+An additional container holds each button block (one or more buttons), and has
+a class of `wp-block-buttons`.
+
+If chosen, a button can have only an outline (no fill color), which adds the `is-style-outline`
+class on the `.wp-block-button` element.
+
+Controls allow each button to have custom text color, background color or gradient,
+and border radius.
+
+<Preview>
+  <Story name="core/button" height="100px">
+    {`<div class="wp-block-buttons">
+        <div class="wp-block-button">
+          <a class="wp-block-button__link" href="https:/end wp-themes.com/twentytwenty/" style="border-radius:50px" target="_blank" rel="noopener noreferrer">
+            Button
+          </a>
+        </div>
+    </div>`}
+  </Story>
+</Preview>
+
+## Code
+
+The code block adds the class `wp-block-code` to a `pre` tag wrapping content inside
+`code` tags.
+
+<Preview>
+  <Story name="core/code" height="350px">
+    {`<pre class="wp-block-code">
+        <code>
+          // A "block" is the abstract term used
+          // to describe units of markup that,
+          // when composed together, form the
+          // content or layout of a page.
+          registerBlockType( name, settings );
```

Comment:
I recommend choosing one of the code samples from [our code block story](https://v-next--cloudfour-patterns.netlify.com/?path=/story/design-typography--code-block) instead of on that is 90% comment. (You can view different examples via a knob.)

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/590#discussion_r405779536